modules/fontconfig/hm.nix

```diff
@@ -0,0 +1,12 @@
+{ mkTarget, ... }:
+mkTarget {
+  name = "fontconfig";
+  humanName = "Fontconfig";
+
+  configElements =
+    { fonts }:
+    {
+      fonts.fontconfig.enable = true;
```

Comment:
I think this is contrary to the rest of stylix. But this is existing behavior and I did not intend to change behavior with this PR.

I'd be happy to follow up with a removal of this and rebasing of #1292.

Reply:
Resolving because approved https://github.com/nix-community/stylix/pull/1381#pullrequestreview-2868937842